CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Changelog
 
 ## [Unreleased]
+### Changed
```

Comment:
My assumption was that the person with npm publishing credentials would decide.
I'll add it anyway.